docs/tutorials/model_to_instance.ipynb

```diff
@@ -0,0 +1,314 @@
+    "\\mathrm{s.t.} \\quad & \\sum_{i=0}^{n-1} w_i x_i \\leq W, \\\\\n",
+    "& x_{i} \\in \\{ 0, 1\\} \n",
+    "\\end{align*}\n",
+    "$$\n",
+    "\n",
+    ":::{note}\n",
+    "ナップサック問題の定式化について詳しく知りたい場合は [こちら](https://www.documentation.jijzept.com/docs/tutorial/knapsack/) を参照してください。\n",
+    ":::\n",
+    "\n",
+    "この数理モデルにあるそれぞれの文字の意味は以下の通りです：\n",
+    "\n",
+    "| 文字 | 説明 |\n",
+    "| --- | --- |\n",
+    "| $N$ |\tアイテムの総数 |\n",
+    "| $v_{i}$ | アイテム $i$ の価値 |\n",
+    "| $w_{i}$ | アイテム $i$ の重さ |\n",
+    "| $W$ | ナップサックの耐荷重 |\n",
+    "\n",
+    "この数理モデルの係数 $v_{i}, w_{i}, W$ に、次のインスタンスデータを入力して得られるインスタンスをソルバーで解くことを考えます：\n",
+    "\n",
+    "\n",
+    "| 文字 | 説明 |\n",
+    "| --- | --- |\n",
+    "| $\\boldsymbol{v}$ | `[10, 13, 18, 31, 7, 15]` |\n",
+    "| $\\boldsymbol{w}$ | `[11, 15, 20, 35, 10, 33]` |\n",
+    "| $W$ | `47` |\n",
+    "\n",
+    "JijModelingを使うと、ソルバーに入力するためのインスタンスを次の3ステップで生成できます：\n",
+    "\n",
+    "1. JijModelingでナップサック問題を定式化する\n",
+    "2. `Interpreter` オブジェクトにインスタンスデータを登録する\n",
+    "3. `Interpreter` オブジェクトを使って数理モデルをインスタンスに変換する\n",
+    "\n",
+    "![Diagram of the process to generate an instance from a mathematical model](./assets/model_to_instance_01.png)\n",
+    "\n",
+    "## Step1. JijModelingでナップサック問題を定式化する\n",
+    "\n",
+    "JijModelingを使用してナップサック問題を定式化すると、以下のPythonコードになります："
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/latex": [
+       "$$\\begin{array}{cccc}\\text{Problem:} & \\text{problem} & & \\\\& & \\max \\quad \\displaystyle \\sum_{i = 0}^{N - 1} v_{i} \\cdot x_{i} & \\\\\\text{{s.t.}} & & & \\\\ & \\text{重量制限} & \\displaystyle \\sum_{i = 0}^{N - 1} w_{i} \\cdot x_{i} \\leq W &  \\\\\\text{{where}} & & & \\\\& x & 1\\text{-dim binary variable}\\\\\\end{array}$$"
+      ],
+      "text/plain": [
+       "<jijmodeling.Problem at 0x560c949d7010>"
+      ]
+     },
+     "execution_count": 1,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "import jijmodeling as jm\n",
+    "\n",
+    "# アイテムの価値\n",
+    "v = jm.Placeholder(\"v\", ndim=1)\n",
+    "# アイテムの重さ\n",
+    "w = jm.Placeholder(\"w\", ndim=1)\n",
+    "# ナップサックの耐荷重\n",
+    "W = jm.Placeholder(\"W\")\n",
+    "# アイテムの総数\n",
+    "N = v.len_at(0, latex=\"N\")\n",
+    "# アイテムiをナップサックに入れる場合は1, 入れない場合は0を取る決定変数\n",
+    "x = jm.BinaryVar(\"x\", shape=(N,)) \n",
+    "# アイテムに割り当てられた番号を走る添え字\n",
+    "i = jm.Element(\"i\", belong_to=(0, N))\n",
+    "\n",
+    "problem = jm.Problem(\"problem\", sense=jm.ProblemSense.MAXIMIZE)\n",
+    "# 目的関数\n",
+    "problem += jm.sum(i, v[i] * x[i])\n",
+    "# 制約条件: ナップサックの耐荷重を超えない\n",
+    "problem += jm.Constraint(\"重量制限\", jm.sum(i, w[i] * x[i]) <= W)\n",
+    "problem"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Step2. Interpreter オブジェクトにインスタンスデータを登録する\n",
```

Comment:
```suggestion
    "## Step2. `Interpreter` オブジェクトにインスタンスデータを登録する\n",
```

Reply:
fix in https://github.com/Jij-Inc/JijModeling-Tutorials/pull/6/commits/c850ac4bf6d5f7f971ea6ccda91e1b02c17c849b